File: programacao/c/respostas/selecao/08.c
#include <stdio.h>

int main(void) {
    float saldo, valor;

    printf("Digite o saldo inicial: ");
    scanf("%f", &saldo);
    printf("Digite o valor da operacao: ");
    scanf("%f", &valor);

    if (valor < 0) {
        if (saldo + valor >= 0) {
            printf("Saldo final: %.2f\n", saldo + valor);
        } else {
            printf("Saldo insuficiente\n");
        }
    } else {
        printf("Saldo final: %.2f\n", saldo + valor);
    }

    return 0;
}

File: programacao/python/respostas/selecao/08.py
saldo = float(input("Digite o saldo inicial: "))
valor = float(input("Digite o valor da operacao: "))

if valor < 0:
    if saldo + valor >= 0:
        print(f"Saldo final: {saldo + valor:.2f}")
    else:
        print("Saldo insuficiente")
else:
    print(f"Saldo final: {saldo + valor:.2f}")
